docs/release-process.md

```diff
@@ -0,0 +1,184 @@
+# How we release c2pa-rs and related crates
+
+The build process for the crates in this repo is automated using [release-plz](https://release-plz.dev), a Rust-specific CI tool.
+
+Release-plz leverages information contains in the various `Cargo.toml` files and its own [`release-plz.toml`](../release-plz.toml) configuration file. 
+
+There are many options available for [configuring release-plz](https://release-plz.dev/docs/config). In this document, we will talk only about how it is configured for this project. For more detail about how release-plz works, we recommend reading the [release-plz documentation site](https://release-plz.dev/docs), which is quite extensive.
```

Comment:
I would remove that sentence: `There are many options available for [configuring release-plz](https://release-plz.dev/docs/config).`. If it's to link to the docs, maybe put the link on the tool name?

Reply:
Good call. Simplified.